rust/src/shards/gui/widgets/listbox.rs

```diff
@@ -131,7 +144,26 @@ impl Shard for ListBox {
       }
     }
 
-    Ok(common_type::any)
+    let input_type = data.inputType;
+    let slice = unsafe {
+      let ptr = input_type.details.seqTypes.elements;
+      std::slice::from_raw_parts(ptr, input_type.details.seqTypes.len as usize)
+    };
+    let element_type = if !slice.is_empty() {
+      slice[0]
```

Comment:
this is not safe, you should check this way:
If len is 1 ok to use slice[0]
If len is >1 use any cos we cannot be sure

Reply:
I could also check all types if `len > 1` and return ` any` only if two types are different.

Reply:
Type info for seq should not include duplicates, if it does we do something wrong 😅 you can assert that if you want